common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -361,6 +361,65 @@ def test_showanswer_past_due(self):
                                             graceperiod=self.two_day_delta_str)
         self.assertFalse(still_in_grace.answer_available())
 
+    def test_showanswer_after_attempts(self):
+        """
+        With showanswer="after_attempts",
+        the show answer button should
+        only appear after the user has
+        attempted the problem(i.e clicked the
+        submit button) for the requisite number of times.
+
+        Once the button appears, it does not disappear again.
+        """
+        # Cannot see answer because user has not attempted
```

Comment:
These should actually just be separate test cases.

Each test should generally only assert one thing.
With proper naming, the comments won't even be necessary.
Or if it makes sense to keep the comments, they should be docstrings to the individual methods.